Makefile

```
@@ -135,18 +135,18 @@ env:
 
 $(INITIAL_BUILD): $(SOURCE_ID)
 	@echo "# Running tests..." 1>&2
-	@$(RUN_TESTS_QUIET)
-	@BIN_PATH="$@" ./dev/build initial > /dev/null
+	@$(RUN_TESTS_QUIET) || exit 1
```

These `... || exit 1` lines are to ensure that we fail early when appropriate.